From 2f4bb1323012f8506188ec88b93ddfe2f34cf1a0 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Wed, 12 Sep 2018 17:29:24 +0000
Subject: [PATCH] OS-7242 $(VARPD_MAPFILES) escaped out of the varpd makefiles
 Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com> Reviewed by: Patrick
 Mooney <patrick.mooney@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 usr/src/lib/librename/Makefile | 2 +-
 usr/src/lib/libsff/Makefile    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/lib/librename/Makefile b/usr/src/lib/librename/Makefile
index 222523d9a1..1278c6cc9d 100644
--- a/usr/src/lib/librename/Makefile
+++ b/usr/src/lib/librename/Makefile
@@ -30,7 +30,7 @@ lint :=		TARGET = lint
 
 all clean clobber lint: $(SUBDIRS)
 
-install: 	$(SUBDIRS) $(VARPD_MAPFILES) install_h
+install: 	$(SUBDIRS) install_h
 
 install_h: 	$(ROOTHDRS)
 
diff --git a/usr/src/lib/libsff/Makefile b/usr/src/lib/libsff/Makefile
index 39c13c30bf..134bf99c6d 100644
--- a/usr/src/lib/libsff/Makefile
+++ b/usr/src/lib/libsff/Makefile
@@ -30,7 +30,7 @@ lint :=		TARGET = lint
 
 all clean clobber lint: $(SUBDIRS)
 
-install: 	$(SUBDIRS) $(VARPD_MAPFILES) install_h
+install: 	$(SUBDIRS) install_h
 
 install_h: 	$(ROOTHDRS)
 
-- 
2.21.0

